README.md

```diff
@@ -214,7 +215,29 @@ class RPCHandler(object):
         pass
 ```
 
+### Disable for testing
 
+When running tests you usally need to mock the relayer so it doesn't try to connect to kafka while running the tests, to ease that relayer includes the `relayer.test` package to help you mock it.
+
+Example test file:
+
+```python
+from unittest import TestCase
+
+from relayer.test import RelayerPatch
+
+class MyTestCase(TestCase):
+
+    def setUp(self):
+        self.relayer_patch = RelayerPatch()
```

Comment:
That's why it's created on `setUp`, why would a `create_app` exists before the `setUp` call ?